src/popup/components/buttons/BtnAddPayload.vue

```diff
@@ -0,0 +1,50 @@
+<template>
```

Comment:
This button can be used in more than one place (Add payload). I suggest to set more generic name.